test/lit.cfg

```diff
@@ -708,6 +708,10 @@ elif swift_test_mode == 'only_executable':
     config.limit_to_features.add("executable_test")
 elif swift_test_mode == 'only_non_executable':
     config.available_features.add("nonexecutable_test")
+elif swift_test_mode == 'with_cxx_interop':
+    config.available_features.add("with_cxx_interop")
+    config.swift_frontend_test_options += ' -enable-cxx-interop'
```

Comment:
Can we use `-enable-expiremental-cxx-interop` here and below? I'm trying to get rid of `enable-cxx-interop`.